src/platform/packages/shared/kbn-discover-utils/src/field_constants.ts

```diff
@@ -44,3 +48,15 @@ export const DEGRADED_DOCS_FIELDS = [IGNORED_FIELD, IGNORED_FIELD_VALUES_FIELD]
 export const ERROR_STACK_TRACE = 'error.stack_trace';
 export const ERROR_EXCEPTION_STACKTRACE_ABS_PATH = 'error.exception.stacktrace.abs_path';
 export const ERROR_LOG_STACKTRACE_ABS_PATH = 'error.log.stacktrace.abs_path';
+
+// Traces Fields
+export const PARENT_ID_FIELD = 'parent.id';
+export const TRANSACTION_ID_FIELD = 'transaction.id';
+export const TRANSACTION_NAME_FIELD = 'transaction.name';
+export const TRANSACTION_DURATION_FIELD = 'transaction.duration.us';
+export const SPAN_NAME_FIELD = 'span.name';
+export const SPAN_ACTION_FIELD = 'span.action';
+export const SPAN_DURATION_FIELD = 'span.duration.us';
+export const SPAN_TYPE_FIELD = 'span.type';
+export const SPAN_SUBTYPE_FIELD = 'span.subtype';
+export const SPAN_DESTINATION_SERVICE_RESOURCE_FIELD = 'span.destination.service.resource';
```

Comment:
I just realized that `kbn-apm-types` package is located in the `platform` folder. I think we could get the field names from there instead of duplicating them here.

Reply:
ah. nvm. it's under `x-pack`

Reply:
either way, that's for a follow up PR.